release/models/policy-forwarding/openconfig-pf-network-instances.yang

```diff
@@ -0,0 +1,194 @@
+submodule openconfig-pf-network-instances {
+  belongs-to openconfig-policy-forwarding {
+    prefix "oc-pf";
+  }
+
+  import openconfig-extensions { prefix "oc-ext"; }
+  import openconfig-interfaces { prefix "oc-if"; }
+  import openconfig-network-instance { prefix "oc-ni"; }
+
+  include openconfig-pf-forwarding-policies;
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This submodule contains groupings to attach
+    a policy-forwarding policy to one or more network-instances.
+    
+    When attached to a network-instance, the policy-forwarding rules
+    should be evaluated against packets in the specified
+    network-instances.";
+
+  oc-ext:openconfig-version "0.8.0";
+
+  revision "2024-11-15" {
+    description
+      "Add network-instance as an attachment point.";
+    reference "0.8.0";
+  }
+
+  revision "2024-11-14" {
+    description
+      "Clarify that if no rules are present, no packets will be matched.";
+    reference "0.7.0";
+  }
+
+  revision "2023-04-25" {
+    description
+      "Clarify use of interface-ref.";
+    reference "0.6.1";
+  }
+
+  revision "2023-03-27" {
+    description
+      "Add support for decap in one NI and further actions in other NIs.";
+    reference "0.6.0";
+  }
+
+  revision "2022-01-25" {
+    description
+      "Add GUE and MPLS-in-UDP decapsulation actions.";
+    reference "0.5.0";
+  }
+
+  revision "2021-08-06" {
+    description
+      "Update path to the network instance action to allow references
+      to other NIs.";
+    reference "0.4.0";
+  }
+
+  revision "2021-05-19" {
+    description
+      "Add a VRF selection policy type, and means to apply the
+      policy.";
+    reference "0.3.0";
+  }
+
+  revision "2018-11-21" {
+    description
+      "Add OpenConfig module metadata extensions.";
+    reference "0.2.1";
+  }
+
+  revision "2017-06-21" {
+    description
+      "Amend policy forwarding model based on ACL changes.";
+    reference "0.2.0";
+  }
+
+  revision "2017-02-28" {
+    description
+      "Initial public release of policy forwarding.";
+    reference "0.1.0";
+  }
+
+  revision "2016-11-08" {
+    description
+      "Initial revision";
+    reference "0.0.1";
+  }
+
+
+  grouping pf-network-instance-structural {
```

Comment:
@dplore, I couldn't see where this grouping is used, is part of the change set missing?

Reply:
yes, this a a draft PR / work in progress.  I'll switch it to "ready to review" shortly